src/patterns/components/highlight-board/react/code/default.hbs

```diff
@@ -0,0 +1,8 @@
+<SprkHighlightBoard
+  imgSrc="https://staging.sparkdesignsystem.com/assets/toolkit/images/desktop.jpg"
+  imgAlt='flower'
+  heading='Hello, Welcome To Spark Design System'
+  ctaText='Button 1'
+  ctaHref='#'
+  ctaText2='Button 2'
+  ctaHref2='#'/>
```

Comment:
Hey @bobwise, these React Code examples need to match the content that's displayed in the site. So, when they are used - they resemble the rendering we have in the doc site. Button 1 and Button 2 / alt text, ETC ETC